content/efcore/getting-started/samples/defining-entity-relationships/many-to-many-relationship/README.md

```diff
@@ -0,0 +1,12 @@
+# Many-to-Many Relationships
+
+This code example demonstrates how to create a many-to-many relationship using a joining table.
+
+## Code Files:
+- [Program.cs](../shared/Program.cs)
+- [Reader.cs](../../shared/Reader.cs)
+- [CheckoutRecord.cs](../../shared/CheckoutRecord.cs)
+- [Book.cs](../../shared/Book.cs)
+- [Address.cs](../../shared/Address.cs)
+- [Author.cs](../../shared/Author.cs)
+- [LibraryContext.cs](LibraryContext.cs)
```

Comment:
Make `LibraryContext.cs` the second item in the list so that it is one of the first files to appear in the console.